### PR TITLE
add "current" class to number span in view; use storybook knobs

### PR DESCRIPTION
**Pull request by crazy4groovy**
This can help developers style the "current" number, and hide the others above/below it (eg. if there's top padding). Possibly via a transitioned opacity.

Also, I think the Knobs plugin is pretty sweet.

### CODE REVIEW

**Comment by crazy4groovy** [comment]
```css
span.text {
 transition: opacity 1s;
}
span.text.currentValue {
 opacity: 1
}
span.text.hiddenValue {
 opacity: 0
}
```

**Comment by browniefed** [comment]
Hey, thanks for the PR. I'll review this a little later.
Rather than defining `currentValue` and `hiddenValue` as specific class names these should be injected as props so that these work with any CSS-in-JS, cssmodules, etc libraries.

**Comment by crazy4groovy** [comment]
Sure thing! I've added these classes as optional props, also cleaned up the code with "class properties" for this.state initiation.

File: src/index.js [hunk 1]
```diff
@@ -26,31 +26,36 @@ const pieceStyle = {
   zIndex: "10",
 };
 
-class Tick extends Component {
-  render() {
-    const { value, height, range, className } = this.props;
-    const index = range.indexOf(value);
+const Tick = props => {
```
Comment (crazy4groovy) [inline src/index.js]: KISS, using a function component

**Comment by crazy4groovy** [comment]
^^ poke?

**Comment by crazy4groovy** [comment]
Can you review this PR? Thanks

**Pull request merged**